docker/dragonfly2/tests/functional_tests/scheduler_test.yaml

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Add Apache 2 Header here and other template files. You have the right header in versions.yaml